### PR TITLE
Tc/org2

### PR DESCRIPTION
**Pull request by CedThen**

### CODE REVIEW

**Comment by showduhtung** [comment]
what is org2

**Comment by showduhtung** [comment]
Screen Shot 2021-05-13 at 3 28 12 PM
really fixed did you now

File: server/utilities/helperFunctions.js [hunk 1]
```diff
@@ -392,7 +392,7 @@ export const updateEvents = async (events, t) => {
           parentSchedule.end,
           parentService.day,
         );
-        taskDays.forEach((date) =>
+        taskDays.forEach(async (date) =>
```
Comment (showduhtung) [inline server/utilities/helperFunctions.js]: interesting that this was a fix. I linked you an article that says this isn't necessary but.. hmm i dunno!

**Comment by showduhtung** [comment]
this branch looks like it's branched off of the other one. it's hard to do 2 pull requests that are dependent on each other. is it possible to have done this separately?

File: server/utilities/helperFunctions.js (hunk 1 above)
Comment (CedThen) [reply server/utilities/helperFunctions.js]: seems to work without the async haha

**Review by showduhtung — APPROVED** [review]

**Pull request merged**